internal/services/dashboard/dashboard_grafana_resource.go

```diff
@@ -89,6 +107,70 @@ func (r DashboardGrafanaResource) Arguments() map[string]*pluginsdk.Schema {
 			}, false),
 		},
 
+		"configuration": {
```

Comment:
i don't think we need this block? configuration is implied?  could we remove this and bring up smtp a level?

Reply:
Will do.